docs/getting-started/README.md

```diff
@@ -1,31 +1,31 @@
 
-Frontity also welcomes contributions. There are many ways to support the project and get involved. Plus, you can get **free swag**! Please refer to the guide below to learn more about it:
+* **Node installed in your computer** - If you don't already have it you can get **Node** from [the official site](https://nodejs.org/). Alternatively, you can install Node using **nvm** \([node version manager](https://github.com/creationix/nvm)\). Any of these methods will also install **npm** and **npx** along with Node. You will use these to run Frontity commands during the set-up and development of your project.
 
-* 🙌 [**How to contribute**](../contributing/how-to-contribute.md)
+> For those coming from WordPress it might be worth noting that **Frontity** runs on **Node**, so it needs to be deployed in a different server than your WordPress. If you want to learn more about this, visit our [GitHub repo](https://github.com/frontity/frontity#why-a-different-nodejs-server) or see the [Architecture](architecture/README.md) section of these docs.
 
-## How is Frontity organized?
+Once you have a WordPress site and Node you're good to go and all set to get up and running....
 
-There are different channels where you can find information about the project and discuss about it:
+## Get up and running
 
-* **📖  Docs:** this is the place to learn how to build amazing sites with Frontity. [Get started](quick-start-guide.md)!
-* **👨‍👩‍👧‍👦  Community forum:** use our forum to ask any questions, share feedback and meet great people. This is your place too to share [how are you planning to use Frontity](https://community.frontity.org/c/dev-talk-questions)!
-* **🐞  GitHub:** we use _\*\*_[GitHub](https://github.com/frontity/frontity) for bugs and pull requests, questions are answered in the community forum.
-* **🗣  Social media**: a more informal place to interact with Frontity users. Reach out to us on [Twitter](https://twitter.com/frontity).
-* 💌  **Newsletter:** do you want to receive the latest framework updates and news? Subscribe [here](https://frontity.org/#newsletter).
+No time to waste? Then head straight over to our [**Quick start guide**](quick-start-guide.md) to get up and running right away.
+
+Want all the gen before you get started? Then read through our [**Learning Frontity**](learning-frontity/README.md) section for in-depth insights into working with Frontity.
+
+Need more info? Our [**Community forum**](https://community.frontity.org/) is the perfect place to ask any questions and share your feedback. We look forward to seeing you there.
+
+Frontity is open source and welcomes contributions. There are many ways to support the project and get involved. Plus, you can get **free swag**! Please see the [**Contributing**](contributing/README.md) section for more info.
```

Comment:
Let's confirm with @Poliuk what is this thing about 
> Plus, you can get **free swag**!
before letting it in the docs

Reply:
That was in the original so I left it in. TBH I'm inclined to take it out completely. Free swag should not be the motivation for contributing to open source.

Reply:
I agree with you @mburridge, remove it.

Reply:
Removed reference to free swag

Reply:
The change is not reflected in this PR

Reply:
❓
https://github.com/frontity/gitbook-docs/pull/32/commits/7d3467765655f9033239177c560383fae42d92de